test/__init__.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+formatter = logging.Formatter("%(levelname)s:%(name)s:%(message)s")
+
+stream_handler = logging.StreamHandler()
+stream_handler.setFormatter(formatter)
+
+logger = logging.getLogger("hca-matrix-service-testcase")
+logger.addHandler(stream_handler)
```

Comment:
Here too, I would just use the standard logging configuration as outlined in the AWS doc.

Reply:
fixed.